config/default.js

```diff
@@ -37,4 +37,13 @@ module.exports = {
     filename: './db.sqlite3',
     removeBeforeOpen: false,
   },
+  authentication: {
+    enabled: false,
+    defaultUser:  {              // DO NOT USE THIS IN PRODUCTION
+      email: 'user@example.com',
+      password: 'password',
+      name: 'Example User'
+    },
+    secret: 'top secret 51'      // DO NOT USE THIS IN PRODUCTION
+  }
 };
```

Comment:
At the moment I think both would be the same anyway. These values have to be provided but we want authentication disabled by default. The follow up bugs will implement the features which allow these to be set dynamically, at which point it makes sense to remove this from the default config and a separate development.js and production.js is a great idea.